plugins/api-docs/README.md

```diff
@@ -219,6 +219,24 @@ security:
     - [read_pets, write_pets]
 ```
 
+### Adding requestInterceptor to Swagger UI
+
+To configure a [requestInterceptor for Swagger UI](https://github.com/swagger-api/swagger-ui/tree/master/flavors/swagger-ui-react#requestinterceptor-proptypesfunc) you'll need to add the following to your `App.tsx`:
+
+```tsx
+...
+import { apiDocsPlugin, ApiExplorerPage } from '@backstage/plugin-api-docs';
+...
+apiDocsPlugin.__experimentalReconfigure({
```

Comment:
The `__experimentalReconfigure` experiment hasn't panned out like we hoped and it doesn't work well with future plans either. We plan to remove it but are somewhat dependent on i18n to be added as that's what it's used for right now. Basically we don't want to add any more usage of it.

What I think would make sense instead in this case is to add the request interceptor to the available props [here](https://github.com/backstage/backstage/blob/de39377ee7994de780d9cfc030b714967c3525ef/plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinition.tsx#L137-L139). That way you can provide your own implementation of the `ApiDocsConfig` API with a customized widget.